pom.xml

```diff
@@ -11,7 +11,7 @@
     <url>https://github.com/IBM/secrets-manager-java-sdk</url>
 
     <groupId>com.ibm.cloud</groupId>
-    <version>2.0.15</version>
+    <version>2.0.16</version>
```

Comment:
This change should be removed. Publication automation bumps version automatically.